psc-ide-flycheck.el

```diff
@@ -134,7 +134,7 @@ CALLBACK is the status callback passed by flycheck."
 (flycheck-define-generic-checker 'psc-ide
   "A purescript syntax checker using the `psc-ide' interface."
   :start #'psc-ide-flycheck-start
-  :modes '(purescript-mode))
+  :modes (list psc-ide-major-mode-symbol))
```

Comment:
Using some literal escaped list e.g. `'(,psc-ide-major-mode-symbol)` does not work here, since whatever `:modes` arguments are fed to some macro